extensions/notebook/src/book/bookTreeView.ts

```diff
@@ -76,6 +76,15 @@ export class BookTreeViewProvider implements vscode.TreeDataProvider<BookTreeIte
 		if (bookPathToTrust) {
 			let trustChanged = this._bookTrustManager.setBookAsTrusted(bookPathToTrust);
 
+			// update trust state of opened items
+			if (azdata.nb.notebookDocuments && azdata.nb.notebookDocuments.length) {
```

Comment:
Any reason why you're checking the length here? If it's empty the forEach won't do anything so this is unnecessary.

Reply:
# resolved